src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -941,7 +975,37 @@ private PhysicalLocation CreatePhysicalLocation(string uri, Region region = null
                 Region = region,
                 ContextRegion = region
             };
+        }
+
+        private static readonly Regex LogicalLocationRegex =
+            new Regex(
+                @"
+                  ([^\s]*\s+)?         # Skip over an optional leading blank-terminated return type name such as 'void '
```

Comment:
>         # Skip over an optional leading blank-terminated return type name such as 'void ' [](start = 30, length = 90)

wow are comments supported in these regexes? if so, looks like once you introduce a # character the remainder of the line is treated as a comment? TIL

Reply:
Yes. Super-helpful for complicated regexes.

---
In reply to: [282150310](https://github.com/microsoft/sarif-sdk/pull/1480#discussion_r282150310) [](ancestors = 282150310)